clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -172,3 +272,14 @@ type target struct {
 	Labels map[string]string `json:"labels"`
 	Health string            `json:"health"`
 }
+
+func getKubemarkMasterClusterIp() (ip string, err error) {
```

Comment:
Why do use named return values if there are not used in the code?

Reply:
This was a leftover from a previous version. Removed, thanks!